tools/topology/topology2/cavs-mixin-mixout-fir-hda.conf

```diff
@@ -0,0 +1,119 @@
+Define {
+	ANALOG_PLAYBACK_PCM		'Analog Playback'
+	ANALOG_CAPTURE_PCM		'Analog Capture'
+	HDA_ANALOG_DAI_NAME      	'Analog Playback and Capture'
+	DEEP_BUFFER_PIPELINE_ID		15
+	DEEP_BUFFER_PCM_ID		31
+	DEEP_BUFFER_PIPELINE_SRC	'mixin.15.1'
+	DEEP_BUFFER_PIPELINE_SINK	'mixout.2.1'
+	DEEP_BUFFER_PCM_NAME		'Deepbuffer HDA Analog'
+}
+
+# include deep buffer config if buffer size is in 1 - 1000 ms.
+IncludeByKey.DEEPBUFFER_FW_DMA_MS{
+	"[1-1000]" "platform/intel/deep-buffer.conf"
+}
+
+Object.Dai {
+	HDA.0 {
+		name $HDA_ANALOG_DAI_NAME
+		id 4
+		default_hw_conf_id 4
+		Object.Base.hw_config.HDA0 {}
+		direction duplex
+	}
+}
+Object.Pipeline {
+	mixout-gain-fir-dai-copier-playback.0 {
```

Comment:
The only difference between this one and the cavs-mixin-mixout-hda.conf is what pipeline is used in the backend. Can we make this conditional based on a variable definition?

Reply:
there's also the byte control name definition... so diff is still small, but not only conf name. I can try to add some conditional logic.